catalog/core/catalog-core-standardframework/pom.xml

```diff
@@ -137,6 +137,7 @@
         <dependency>
             <groupId>com.hazelcast</groupId>
             <artifactId>hazelcast</artifactId>
+            <version>${hazelcast.version}</version>
```

Comment:
❗️ We shouldn't add the version here. We should leave the version tag off, so that it inherits from the `dependencyManagement` section that defines the version in the parent pom:
https://github.com/codice/ddf/blob/5674ce0e908704933d3fc51b384757847f8a1262/catalog/pom.xml#L429-L433